flight/Modules/Battery/battery.c

```diff
@@ -226,9 +228,18 @@ static void batteryTask(void * parameters)
 			float alpha = 1.0f - dT / (dT + 2.0f);
 			flightBatteryData.AvgCurrent = alpha * flightBatteryData.AvgCurrent + (1 - alpha) * flightBatteryData.Current; //in Amps
 
+			// XXX Arguably this should be to 10% capacity or 15%
```

Comment:
Question is, should the time read 0 at 80% discharge?  Who does the 80%?  Probably us, but not sure.